base/datasets.go

```diff
@@ -9,14 +9,26 @@ import (
 )
 
 // ListDatasets lists datasets from a repo
-func ListDatasets(r repo.Repo, limit, offset int, RPC bool) (res []repo.DatasetRef, err error) {
+func ListDatasets(r repo.Repo, limit, offset int, RPC, publishedOnly bool) (res []repo.DatasetRef, err error) {
```

Comment:
About this filename, there's only one function here, `ListDatasets`. Can it be moved into the very similarly named file "dataset.go"? Or vice versa?